test/topology/test_change_ip.py

```diff
@@ -8,56 +8,118 @@
 """
 
 import logging
+import time
+
 import pytest
-from test.pylib.util import wait_for_cql_and_get_hosts, wait_for
+import uuid
+from cassandra.pool import Host  # type: ignore # pylint: disable=no-name-in-module
+from test.pylib.internal_types import ServerInfo
 from test.pylib.random_tables import Column, IntType, TextType
-import time
+from test.pylib.util import wait_for_cql_and_get_hosts, wait_for
+
 logger = logging.getLogger(__name__)
 
 
 @pytest.mark.asyncio
-async def test_change_two(manager, random_tables):
+async def test_change_two(manager, random_tables, mode):
```

Comment:
Please execute the test 100 times in debug mode to eliminate potential flakiness

Reply:
got two failures out of 100, will try to investigate tomorrow

Reply:
fixed